web/src/views/FindSpeaker.vue

```diff
@@ -86,21 +90,24 @@ export default {
     Search,
     SpeakerCard,
     PaginationRow,
+    NoResults,
+  },
+  data() {
```

Comment:
(minor comment) There seem to be different opinions on which syntax is preferable. Actually this way looks pretty nice, though some of the VS Code plugins I use feel differently. 😄